src/content/en/updates/2018/09/inside-browser-part2.md

```diff
@@ -114,10 +114,13 @@ they need to pass the data to download manager.
 
 This is also where the [SafeBrowsing](https://safebrowsing.google.com/) check happens. 
 If the domain and the response data seems to match a known malicious site, then the network thread 
-alerts to display a warning page. Additionally, 
+alerts to display a warning page. 
+
+Note: this is a check point every network request (both page navigation and subresource download) 
```

Comment:
@ericlaw1979 I edited the part about CORB per your comment on the post, WDYT ?

Reply:
Grammatically, I think a word is missing. Maybe:

"Note: this is a **checkpoint on** every network request (both page navigation and subresource download)"?

Reply:
+1